.github/workflows/change-plugin-scope.yml

```diff
@@ -0,0 +1,113 @@
+# Description:
+# Change plugin scope.
+# It uses action grafana/plugin-ci-workflows/actions/plugins/publish/promote-plugin
+# to change plugin scope to universal.
+# The following inputs should be provided:
+# - environment: Target environment (dev, ops, prod)
+# - scopes: Scopes to change to. Can be 'universal', 'grafana_cloud', 'grafana_cloud_org' or 'grafana_cloud_instance'.
+#           Multiple values can be provided separated by commas.
+# - plugin_id: Plugin ID to change scope
+# - plugin_version: Plugin version to change scope
+#
+
+name: Plugins - Change Plugin Scope
+
+on:
+  workflow_dispatch:
+    inputs:
+      environment:
+        description: 'Target environment'
+        required: true
+        default: 'dev'
+        type: choice
+        options:
+          - dev
+          - ops
+          - prod
+      plugin_id:
+        description: 'Plugin ID to change scope'
+        required: true
+        type: string
+      plugin_version:
+        description: 'Plugin version to change scope'
+        required: true
+        type: string
+      scopes:
+        description: |
+          Scope to change the plugin to.
+          Can be 'universal', 'grafana_cloud', 'grafana_cloud_org' or 'grafana_cloud_instance'.
+          Multiple values can be provided separated by commas.
+        required: false
+        default: 'universal'
+        type: string
+
+jobs:
+  promote:
+    runs-on: ubuntu-latest
+
+    permissions:
+      id-token: write
+      contents: read
+
+    env:
+      # Set defaults for PR triggers, use inputs for manual dispatch
+      ENVIRONMENT: ${{ inputs.environment || 'dev' }}
+      PLUGIN_ID: ${{ inputs.plugin_id || 'hugooshiro-dummy-datasource' }}
+      PLUGIN_VERSION: ${{ inputs.plugin_version || '1.0.5' }}
```

Comment:
Maybe I'm missing something, but I think the version should be required and always provided explicitly as a input?